src/dwarf/Gfind_proc_info-lsb.c

```diff
@@ -549,57 +544,22 @@ dwarf_find_eh_frame_section(struct dl_phdr_info *info)
   Debug (1, "looking for .eh_frame section in %s\n",
          file);
 
-  fd = open(file, O_RDONLY);
-
-  if (fd == -1)
+  rc = elf_map_image (&ei, file);
```

Comment:
It looks like this changed from open/seek/read/close calls to mmap, but I can't think of any reason that won't work